dienst2/middleware.py

```diff
@@ -46,6 +48,22 @@ def process_request(self, request):
         if request.user.is_authenticated:
             if not IAPBackend.can_authenticate(request):
                 logout(request)
+        elif settings.DANGEROUSLY_ALLOW_AUTOLOGIN:
+            # DEBUG: Automatically log in as admin
+            # Create admin user if it doesn't exist
+            if not User.objects.filter(username="admin").exists():
+                user = User.objects.create(
+                    username="admin",
+                    is_active=True,
+                    is_staff=True,
+                    is_superuser=True,
+                    is_admin=True,
+                )
+
+            else:
+                user = User.objects.get(username="admin")
```

Comment:
It would be more efficient (and pythonic?) to write:

```
try:
    user = User.objects.get(username="admin")
except User.DoesNotExist:
    user = User.objects.create_superuser("admin")
```